daemon/daemon_unix.go

```diff
@@ -690,8 +690,21 @@ func initBridgeDriver(controller libnetwork.NetworkController, config *Config) e
 
 	ipamV4Conf = &libnetwork.IpamConf{AuxAddresses: make(map[string]string)}
 
-	nw, nw6List, err := netutils.ElectInterfaceAddresses(bridgeName)
+	nw, nwList, nw6List, err := netutils.ElectInterfaceAddresses(bridgeName)
 	if err == nil {
+		if config.bridgeConfig.FixedCIDR != "" {
+			_, fCIDR, err := net.ParseCIDR(config.bridgeConfig.FixedCIDR)
+			if err != nil {
+				return err
```

Comment:
Can you use `errors.Wrap()` here from github.com/pkg/errors? It's already vendored.